recipe/bld.bat

```diff
@@ -5,7 +5,10 @@ mkdir %LIBRARY_PREFIX%\lib\cmake\tiff\include
 set "MODS=qtbase"
 set "MODS=%MODS%;qtdeclarative"
 set "MODS=%MODS%;qtimageformats"
-set "MODS=%MODS%;qtmultimedia"
+:REM qt-multimedia will be a separate pacakge after 6.7.1
+IF %PKG_VERSION%=="6.7.1" (
+   set "MODS=%MODS%;qtmultimedia"
+)
```

Comment:
this logic doesn't match up with your tests below for windows. 

I think just delete the windows changes.

For windows users that want multi-media, they can install the qt6-multimedia package you made, and they will be off to the transition a version early.